tools/slack/guidelines.md

```diff
@@ -51,6 +51,7 @@ See a list of other popular channels in the [Slack @ TTS](https://docs.google.co
 ## Usage tips
 
 - **[Add yourself to our custom emoji](#custom-emoji).** Add your profile picture as a custom emoji with your name as the alias ("first-last"). This allows the whole TTS community to celebrate your contributions and serves as your introduction to our prolific custom emoji database. Once reviewed, your new emoji will be posted to [#emoji-showcase](https://gsa-tts.slack.com/messages/C0X2T36AY).
+- **[Take a vote using emojis](#slack-vote).** When you have something that you need multiple people to decide on, you can pose a question and then list multiple options with emojis preset for others to react with. You can keep it simple with :redlight: , :yellowlight: and :greenlight: or use any one of the other [custom emojis](#custom emoji) to get creative. It is
```

Comment:
```suggestion
- **[Take a vote using emojis](#slack-vote).** When you have something that you need multiple people to decide on, you can pose a question and then list multiple options with emojis preset for others to react with. You can keep it simple with :redlight: , :yellowlight: and :greenlight: or use any one of the other [custom emojis](#custom emoji) to get creative.
```

Reply:
Ok with linking to [official docs](https://slack.com/help/articles/229002507-Create-a-poll-in-Slack#use-emoji-reactions) rather than explaining here?

```suggestion
- **[Take a vote using emoji](https://slack.com/help/articles/229002507-Create-a-poll-in-Slack#use-emoji-reactions).**
```